### PR TITLE
[CARBONDATA-569] clean up code for carbon-processing module

### PR DESCRIPTION
**Pull request by jackylk**
clean up:

1. remove unused declaration
2. remove unnecessary exception

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/347/

File: processing/src/main/java/org/apache/carbondata/processing/api/dataloader/SchemaInfo.java [hunk 1]
```diff
@@ -98,54 +98,26 @@ public String getSrcDriverName() {
...
-   * @param srcDriverName The srcDriverName to set.
-   */
-  public void setSrcDriverName(String srcDriverName) {
-    this.srcDriverName = srcDriverName;
-  }
-
-  /**
    * @return Returns the srcConUrl.
    */
   public String getSrcConUrl() {
```
Comment (chenliang613) [inline processing/src/main/java/org/apache/carbondata/processing/api/dataloader/SchemaInfo.java]: Please clear all get methods also.

File: processing/src/main/java/org/apache/carbondata/processing/datatypes/GenericDataType.java [hunk 2]
```diff
@@ -44,11 +44,6 @@
   String getName();
```
Comment (chenliang613) [inline processing/src/main/java/org/apache/carbondata/processing/datatypes/GenericDataType.java]: need delete setName() method also.

File: processing/src/main/java/org/apache/carbondata/processing/api/dataloader/SchemaInfo.java (hunk 1 above)
Comment (jackylk) [inline processing/src/main/java/org/apache/carbondata/processing/api/dataloader/SchemaInfo.java]: fixed

File: processing/src/main/java/org/apache/carbondata/processing/datatypes/GenericDataType.java (hunk 2 above)
Comment (jackylk) [inline processing/src/main/java/org/apache/carbondata/processing/datatypes/GenericDataType.java]: fixed

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/353/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/358/

**Pull request closed without merging**